src/cbapi/psc/livequery/models/device_summary.yaml

```diff
@@ -0,0 +1,37 @@
+type: object
+required: [] # TODO(ww): Find out which result fields are required
```

Comment:
Do we know which result fields are req'd? Can we fill this in?

Reply:
As far as I can tell, no fields are actually "required."  The query will run without any terms; it won't actually _return_ anything, but it won't blow up, either.